napari/_qt/layers/qt_image_layer.py

```diff
@@ -7,51 +8,54 @@
     Rendering,
 )
 from .qt_image_base_layer import QtBaseImageControls
+from ...utils.event import Event
 
 
-class QtImageControls(QtBaseImageControls):
+class QtImageControls(QtBaseImageControls, ImageLayerInterface):
     """Qt view and controls for the napari Image layer.
 
     Parameters
     ----------
     layer : napari.layers.Image
-        An instance of a napari Image layer.
+       An instance of a napari Image layer.
```

Comment:
this is a relic from #1040 but I believe this is the wrong indentation as we should use 4 spaces to indent instead of 3

Reply:
fixed!